package/run_sonobuoy_plugin.sh

```diff
@@ -72,9 +71,7 @@ if [[ "${OVERRIDE_BENCHMARK_VERSION}" != "" ]]; then
       --json \
       --log_dir "${LOG_DIR}" \
       --outputfile "${RESULTS_DIR}/etcd.json" 2> "${ERROR_LOG_FILE}"
-  fi
 else
-  if [[ "$(pgrep -f /etcd | wc -l)" -gt 0 ]]; then
     echo "etcd: Using RANCHER_K8S_VERSION=${RANCHER_K8S_VERSION}"
```

Comment:
Same comment as above. This will break in multi-node clusters where etcd is running only one few and we want to run etcd benchmarks only on the nodes running etd

Reply:
We need to test this with rke1 (multi-node with varied roles), rke2, gke/eks and k3s. All scan profiles need to pass before we change this common area.

Reply:
This has been updated always run when k3s is running. This should limit the impact to other distributions.